Reason: qmake includes deprecated-declerations, which causes deprecated warnings
to result in errors. This is something for upstream to solve, so disable.
Upstream: no
diff --git a/mkspecs/common.pri b/mkspecs/common.pri
index 27fa779..67a8b6d 100644
--- a/mkspecs/common.pri
+++ b/mkspecs/common.pri
@@ -15,18 +15,6 @@ QMAKE_TARGET_COPYRIGHT = Copyright (C) 2010 Nokia
     include($${M_BUILD_TREE}/mkspecs/mlocaleconfig.pri)
 }
 
-#disable werrors on windows and freemantle (M_OS_MAEMO5 comes from meegotouchconfig.pri)
-win32 {
-    } else {
-        contains( DEFINES, M_OS_MAEMO5 ) {
-        } else { 
-	    !contains( M_BUILD_FEATURES, no-werror ) {
-	        QMAKE_CFLAGS *= -Werror
-	        QMAKE_CXXFLAGS *= -Werror
-            }
-        }
-    }
-
 mac {
     INCLUDEPATH += include
 }
